app/scripts/views/mixins/password-prompt-mixin.js

```diff
@@ -0,0 +1,73 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * Plugin to display hints for passwords,
+ * hints change as the user types
+ */
+
+define(function (require, exports, module) {
+  'use strict';
+
+  var BaseView = require('views/base');
+  var t = BaseView.t;
+
+  var $ = require('jquery');
+  var Tooltip = require('views/tooltip');
+
+  var TOOLTIP_MESSAGES = {
+    FOCUS_PROMPT_MESSAGE: t('8 characters minimum, but longer if you plan to sync passwords.'),
+    INITIAL_PROMPT_MESSAGE: t('A strong, unique password will keep your Firefox data safe from intruders.'),
+    MORE_INFO_LINK: t(' <a href="" tabindex="-1" target="_blank">More info..</a>'),
```

Comment:
I think this will confuse the translation team (and will probably have other issues such as RTL?).